ocsp_test.go

```diff
@@ -533,3 +533,21 @@ func TestInitOCSPCacheFileCreation(t *testing.T) {
 		t.Error(err)
 	}
 }
+
+func TestCacheServerDisabled(t *testing.T) {
+	_ = os.Setenv(cacheServerEnabledEnv, "false")
+
+	if _, err := buildSnowflakeConn(context.Background(), Config{
```

Comment:
Can extract building sf conn outside of the `if`? Or at least `Config`? Now it's not very readable.